solutions/src/main/java/ru/sberbank/school/task02/Solution.java

```diff
@@ -0,0 +1,30 @@
+package ru.sberbank.school.task02;
+
+import java.math.BigDecimal;
+
+import ru.sberbank.school.task02.exception.ConverterConfigurationException;
+import ru.sberbank.school.task02.exception.WrongSymbolException;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.Symbol;
+
+public class Solution {
+    public static void main(String[] args) {
+
+        CalculatorFactory calculatorFactory = new CalculatorFactory();
+        // получаем наш валютный калькулятор
+        FxConversionService calculator =
+            calculatorFactory.getFxConversionService(new ExternalQuotesServiceDemo());
```

Comment:
&#39;calculatorFactory&#39; has incorrect indentation level 12, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a403985175>*